third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/call_acquires.move

```diff
@@ -0,0 +1,29 @@
+module 0x8675309::M {
```

Comment:
Same here -- why have these tests been added to reference-safety tests? They do not seem to belong here.

We have organized tests in different directories for a reason.

Reply:
This is from the v1 reference safety tests, and is not ported yet. It contains "acquires" in the name, but is actually a reference safety test.